src/features/views/components/ViewColumnDialog/ColumnGallery.tsx

```diff
@@ -18,34 +21,97 @@ const ColumnGallery: FunctionComponent<ColumnGalleryProps> = ({
   onConfigure,
 }) => {
   const intl = useIntl();
+  const isMobile = useMediaQuery((theme: Theme) =>
+    theme.breakpoints.down('sm')
+  );
+
+  const categoriesRef = useRef<HTMLDivElement>();
```

Comment:
The name `categoriesRef` is a bit misleading IMO, because it's used for the container that doesn't just contain a list of categories, but all of the choice cards. Do you agree?

Maybe something like `contentRef` or `choiceContainerRef` or something would be more clear?

Reply:
i agree!